crates/router/src/core/payouts.rs

```diff
@@ -1930,22 +1973,72 @@ pub async fn update_retrieve_payout_tracker<F, T>(
     Ok(())
 }
 
+/*
+ */
+fn connector_mandate_id_is_none(
```

Comment:
```suggestion
fn should_create_connector_transfer_method(
```

And move this to payout's helpers module